content/updates/quality/3-Spell-check-improvements.md

```diff
@@ -0,0 +1,31 @@
+# Spell-check improvements
+
+## On text area attributes with rich text editor enabled
+::: meta-data type="Improvement" ee-only="true" link-to-doc="../articles/improve-data-quality.html#improve-data-quality"
+:::
+
+To help reach higher data quality content, spell-check and suggestions are now available on text area attributes with rich text editor enabled (WYSIWYG).
+
+## On attribute and option labels
+::: meta-data type="New" ee-only="true" link-to-doc="../articles/improve-data-quality.html#improve-data-quality"
+:::
+
+Now, you can also benefit from spell-checking on attributes and option labels to guarantee the consistency on your customer-facing product data.
+
+You can find this improvement under the attribute and attribute option setting pages. It is also available in the product form. New recommendations are listed, should we find errors. 
```

Comment:
under? I believe it should be ON?

Reply:
Not sure I understand the difference between the two meanings.

Reply:
same here :)

Reply:
Fixed!